chainregistry.go

```diff
@@ -250,6 +262,7 @@ func newChainControlFromConfig(cfg *config, chanDB *channeldb.DB,
 			bitcoindPass, cfg.BitcoindMode.ZMQPath,
 			time.Millisecond*100)
 		if err != nil {
+			fmt.Println("weks!")
```

Comment:
Reminder to remove during rebase :)

Reply:
Already done in 98d28d9, but I'll squash that in. :)